konflux-ci/namespace-lister/certificate.yaml

```
@@ -0,0 +1,18 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: namespace-lister
+  namespace: namespace-lister
+spec:
+  isCA: true
+  subject:
+    organizations:
+      - konflux
+  dnsNames:
+  - localhost
+  - namespace-lister.namespace-lister
```

I think these need to be changed.
```suggestion
  - namespace-lister.namespace-lister.svc
  - namespace-lister.namespace-lister.svc.cluster.local
```